### PR TITLE
Run framework tests on OCM PRs

### PR DESCRIPTION
**Pull request by zyjjay**
ref: https://issues.redhat.com/browse/ACM-4811

### CODE REVIEW

File: .github/workflows/framework-kind.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    tags:
+      - v0.**
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  fw-kind-tests:
+    name: KinD
+    uses: stolostron/governance-policy-framework/.github/workflows/kind.yml@main
```
Comment (dhaiducek) [inline .github/workflows/framework-kind.yml:14]: nit: trailing newline

```suggestion
    uses: stolostron/governance-policy-framework/.github/workflows/kind.yml@main

```

File: .github/workflows/framework-kind.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: Framework
+
+on:
+  push:
+    tags:
+      - v0.**
```
Comment (dhaiducek) [inline .github/workflows/framework-kind.yml:6]: Good idea to run the tests on tag pushes! Is there a reason for this RegEx or could we make it more generic?

```suggestion
      - v[0-9]+.[0-9]+.[0-9]+
```

Comment (zyjjay) [reply .github/workflows/framework-kind.yml:6]: I didn't have anything particular in mind so I just used something that works. I'm open to making it more generic!

**Review by dhaiducek — APPROVED** [review]
Woohoo! Thanks for all your work on this, @zyjjay!

**Pull request merged**